Unable to build release template on windows 11 with error C2440: 'reinterpret_cast': cannot convert from 'XrAction' to 'uint64_t'

### Tested versions

- Reproducible on master branch, commit 1586c5674bbafe1d7f4bd457d446f132b475890c
- Not reproducible on 4.3 stable.

### System information

godot (master branch, commit 1586c5674bbafe1d7f4bd457d446f132b475890c) - Windows 11 (build 26100) - Multi-window, 1 monitor - Vulkan (Forward+) - integrated AMD Radeon(TM) 780M (Advanced Micro Devices, Inc.; 31.0.22048.10001) - AMD Ryzen 7 PRO 7840U w/ Radeon 780M Graphics (16 threads)

### Issue description

I ran the command scons platform=windows target=template_release arch=x86_32, but after some time, an error occurred:

```
scons platform=windows target=template_release arch=x86_32
scons: Reading SConscript files ...
Auto-detected 16 CPU cores available for build parallelism. Using 15 cores by default. You can override it with the `-j` or `num_jobs` arguments.
Using SCons-detected MSVC version 14.3, arch x86_32
Building for platform "windows", architecture "x86_32", target "template_release".
Checking for C header file mntent.h... (cached) no
scons: done reading SConscript files.
scons: Building targets ...
Compiling modules\text_server_adv\text_server_adv.cpp ...
...
many lines...
...
Compiling modules\openxr\extensions\openxr_mxink_extension.cpp ...
←[34mCompiling ←[1mmodules\openxr\extensions\openxr_palm_pose_extension.cpp←[22;23;24;29m ...←[0m♪◙Compiling modules\openxr\extensions\openxr_pico_controller_extension.cpp ...
Compiling modules\openxr\extensions\openxr_valve_analog_threshold_extension.cpp ...
←[34mCompiling ←[1mmodules\openxr\extensions\openxr_visibility_mask_extension.cpp←[22;23;24;29m ...←[0m♪◙Compiling modules\openxr\extensions\openxr_wmr_controller_extension.cpp ...
Compiling modules\openxr\extensions\platform\openxr_vulkan_extension.cpp ...
modules\openxr\openxr_api_extension.cpp(215): error C2440: 'reinterpret_cast': cannot convert from 'XrAction' to 'uint64_t'
modules\openxr\openxr_api_extension.cpp(215): note: Conversion is a valid standard conversion, which can be performed implicitly or by use of static_cast, C-style cast or function-style cast
modules\openxr\openxr_api_extension.cpp(296): error C2440: 'reinterpret_cast': cannot convert from 'XrSwapchain' to 'uint64_t'
modules\openxr\openxr_api_extension.cpp(296): note: Conversion is a valid standard conversion, which can be performed implicitly or by use of static_cast, C-style cast or function-style cast
scons: *** [modules\openxr\openxr_api_extension.windows.template_release.x86_32.obj] Error 2
Compiler terminating.  Please wait........... Abort complete.
scons: *** [modules\text_server_adv\text_server_adv.windows.template_release.x86_32.obj] Error 4
scons: Build interrupted.
scons: building terminated because of errors.
scons: writing .sconsign file.
INFO: Time elapsed: 00:01:48.27
```

By the way, 4.3 stable works fine

### Steps to reproduce

Run `scons platform=windows target=template_release arch=x86_32`

### Minimal reproduction project (MRP)

No project.

--- a/modules/openxr/openxr_api_extension.cpp
+++ b/modules/openxr/openxr_api_extension.cpp
@@ -211,8 +211,8 @@ RID OpenXRAPIExtension::find_action(const String &p_name, const RID &p_action_se
 
 uint64_t OpenXRAPIExtension::action_get_handle(RID p_action) {
 	ERR_FAIL_NULL_V(OpenXRAPI::get_singleton(), 0);
-	XrAction action_hanlde = OpenXRAPI::get_singleton()->action_get_handle(p_action);
-	return reinterpret_cast<uint64_t>(action_hanlde);
+	XrAction action_handle = OpenXRAPI::get_singleton()->action_get_handle(p_action);
+	return (uint64_t)action_handle;
 }
 
 uint64_t OpenXRAPIExtension::get_hand_tracker(int p_hand_index) {
@@ -293,7 +293,7 @@ uint64_t OpenXRAPIExtension::openxr_swapchain_get_swapchain(uint64_t p_swapchain
 
 	OpenXRAPI::OpenXRSwapChainInfo *swapchain_info = reinterpret_cast<OpenXRAPI::OpenXRSwapChainInfo *>(p_swapchain_info);
 	XrSwapchain swapchain = swapchain_info->get_swapchain();
-	return reinterpret_cast<uint64_t>(swapchain);
+	return (uint64_t)swapchain;
 }
 
 void OpenXRAPIExtension::openxr_swapchain_acquire(uint64_t p_swapchain_info) {

